pkg/runctl/config_test.go

```diff
@@ -106,10 +106,10 @@ func Test_loadPipelineRunsConfig_CompleteConfigMap(t *testing.T) {
 	// VERIFY
 	assert.NilError(t, err)
 	expectedConfig := &pipelineRunsConfigStruct{
-		Timeout:       metav1Duration(time.Minute * 4444),
-		NetworkPolicy: "networkPolicy1",
-		LimitRange:    "limitRange1",
-		ResourceQuota: "resourceQuota1",
+		Timeout:                                       metav1Duration(time.Minute * 4444),
+		NetworkPolicy:                                 "networkPolicy1",
+		LimitRange:                                    "limitRange1",
+		ResourceQuota:                                 "resourceQuota1",
```

Comment:
The gofmt check failed, probably due to these lines.

Run `go fmt ./...` using go 1.14.4.

Reply:
I executed this command and pushed but ./build.sh fails locally. gofmt fails locally with this diff:

diff -u pkg/runctl/config_test.go.orig pkg/runctl/config_test.go
--- pkg/runctl/config_test.go.orig  2020-07-21 18:46:50.475572071 +0200
+++ pkg/runctl/config_test.go   2020-07-21 18:46:50.475572071 +0200
@@ -106,10 +106,10 @@
    // VERIFY
    assert.NilError(t, err)
    expectedConfig := &pipelineRunsConfigStruct{
-       Timeout:       metav1Duration(time.Minute * 4444),
-       NetworkPolicy: "networkPolicy1",
-       LimitRange:    "limitRange1",
-       ResourceQuota: "resourceQuota1",
+       Timeout:                                       metav1Duration(time.Minute * 4444),
+       NetworkPolicy:                                 "networkPolicy1",
+       LimitRange:                                    "limitRange1",
+       ResourceQuota:                                 "resourceQuota1",
        JenkinsfileRunnerPodSecurityContextRunAsUser:  int64Ptr(1111),
        JenkinsfileRunnerPodSecurityContextRunAsGroup: int64Ptr(2222),
        JenkinsfileRunnerPodSecurityContextFSGroup:    int64Ptr(3333),

Reply:
`go fmt ./...` removed those spaces in `pkg/runctl/config_test.go` and `gofmt -w .` wants to add those spaces. I'm confused which one should be executed?
When I execute `go fmt ./...` then build.sh will fail and complains that gofmt does not like `pkg/runctl/config_test.go`. And when I run `gofmt -w .` then build.sh will be green.

Reply:
For now I executed `gofmt -w .` and pushed the change to this PR because with this change `build.sh` will be successfull. If you think `go fmt ./...` should be executed then I'd do so.

Reply:
You are using a version of `gofmt` different from the one in Go SDK 1.14.4.